webrtc.html

```diff
@@ -10932,9 +10932,12 @@ <h4>MediaTrackSupportedConstraints, MediaTrackCapabilities,
         <code><a>RTCSessionDescription</a></code> applied via
         <code>setRemoteDescription</code> and the actual RTP data. This means
         that certain members, such as <code>facingMode</code>,
-        <code>echoCancellation</code>, <code>latency</code>,
-        <code>deviceId</code> and <code>groupId</code>, will always be
-        missing.</p>
+        <code>echoCancellation</code>, <code>deviceId</code> and <code>groupId
+        </code>, will always be missing.</p>
+        <p>The <code>latency</code> setting on the audio or video <code>
```

Comment:
Refer to it as a "constrainable property". A setting is a specific value at a particular point in time, not the thing that you are constraining.